packages/ingest/FtpProviderClient.js

```diff
@@ -184,11 +185,11 @@ class FtpProviderClient {
       Body: pass,
       ContentType: lookupMimeType(key)
     };
-    await S3.promiseS3Upload(params);
+    const { ETag: etag } = await S3.promiseS3Upload(params);
     log.info('Uploading to s3 is complete(ftp)', s3uri);
 
     client.destroy();
-    return s3uri;
+    return { s3uri, etag };
```

Comment:
@yjpa7145 Given my other comment about `multipartUploadCopy` perhaps returning `{ ETag }` rather than `{ etag }` (for consistency with the other functions in that module), do we want to do the same here (also for consistency)?

Although I moaned about AWS not using camel case, I think consistency might trump convention here. It might be confusing to change this function (and the others part of this PR) to return on object with a property named `etag`, but everywhere else where we deal with AWS objects, the property is named `ETag`.